src/test/cypress/support/requests/CourseManagementRequests.ts

```diff
@@ -188,11 +188,18 @@ export class CourseManagementRequests {
         const templateCopy = {
             ...modelingExerciseTemplate,
             title,
+        };
+        const dates = {
             releaseDate: dayjsToString(releaseDate),
             dueDate: dayjsToString(dueDate),
             assessmentDueDate: dayjsToString(assessmentDueDate),
         };
-        const newModelingExercise = Object.assign({}, templateCopy, body);
+        let newModelingExercise;
+        if (body.hasOwnProperty('course')) {
+            newModelingExercise = this.getCourseOrExamExercise(Object.assign({}, templateCopy, dates), body);
```

Comment:
Just merge all 3 json objects instead of calling `Object.assign` twice
```suggestion
            newModelingExercise = Object.assign({}, templateCopy, dates, body);
```

Reply:
:+1: